jsk_baxter_robot/jsk_baxter_startup/baxter_rosbag_record.launch

```diff
@@ -0,0 +1,37 @@
+<launch>
+  <arg name="output_prefix" default="baxter" />
+  <arg name="compress" default="false" doc="whether compress rosbag or not." />
+
+  <arg name="realsense_camera_ns" default="realsense_torso" />
+  <arg name="spherical_camera_ns" default="kodak_head" />
+  <arg name="spherical_left_camera_ns" default="elp_head_left" />
+  <arg name="spherical_right_camera_ns" default="elp_head_right" />
```

Comment:
Unlike https://github.com/jsk-ros-pkg/jsk_robot/pull/1572#discussion_r955777593, these arguments seem simple constants.
Please capitalize them and use `value` instead of `default`.